build.gradle.kts

```
@@ -59,7 +59,7 @@ allprojects {
 subprojects {
     tasks.withType<Test>().configureEach {
         predictiveSelection {
-            enabled.set(System.getenv("CI") == null)
+            enabled.set(project.property("enablePTS")?.toString()?.toBooleanStrict() == true)
```

oh, wait a minute, wasn't this lazy before? I think the configuration-cache friend version is something like:
```suggestion
            enabled.set(providers.property("enablePTS").map { it?.toString()?.toBooleanStrict() == true})
```

---

This was extracted from the gradle documentation: https://docs.gradle.com/enterprise/predictive-test-selection/#conditionally_enabling

Seriusly, it is *really* diffcult to implement something in gradle correctly >_<. So many APIs that does the same but only one is the correct.

Fixed.

---

There's a fine line between correctness and easy to understand documentation :) It's a very hard balance. That said it's good feedback for @clayburn ;)